### PR TITLE
Add model implementations and script for running the models

### PR DESCRIPTION
**Pull request by KennethEnevoldsen**
This pull request includes the following commits:

- Delete existing model results (I am unsure if we have been consistent in ensuring that these are up to date)

- Added model implementations and script for running the baseline models

Note: Due to *a very* poor internet connection, there are some changes I haven't made. E.g., I couldn't manage to download the models at the test run of the code and the revision is not correct (HF hub isn't great on a bad connection). Hopefully, it can still get the idea across.

### CODE REVIEW

File: mteb/models/e5_models.py [hunk 1]
```diff
@@ -0,0 +1,63 @@
...
+                if "title" in doc
+                else doc["text"].strip()
+                for doc in corpus
+            ]
+        sentences = ["passage: " + sentence for sentence in sentences]
+        emb = self.mdl.encode(sentences, batch_size=batch_size, **kwargs)
+        return emb
+
+
+e5_mult_small = ModelMeta(
```
Comment (Muennighoff) [inline mteb/models/e5_models.py:56]: Will `e5_mult_small` be used anywhere? Currently, it seems like it's not used, no?

**Review by Muennighoff — COMMENTED** [review]
Looks good to me --- Should I start running?

**Review by isaac-chung — COMMENTED** [review]
Based on the [discussion](https://github.com/embeddings-benchmark/mteb/issues/840#issuecomment-2137388449), I think it's ok for the results to be removed (changed tasks, updated results format etc).

File: mteb/models/e5_models.py [hunk 2]
```diff
@@ -0,0 +1,63 @@
...
+        return self.encode_queries(sentences, batch_size=batch_size, **kwargs)
+
+    def encode_queries(self, queries: list[str], batch_size: int, **kwargs: Any):
+        sentences = ["query: " + sentence for sentence in queries]
+        emb = self.mdl.encode(sentences, batch_size=batch_size, **kwargs)
+        return emb
+
+    def encode_corpus(
+        self, corpus: list[dict[str, str]], batch_size: int, **kwargs: Any
+    ):
```
Comment (Muennighoff) [inline mteb/models/e5_models.py:36]:
```suggestion
    def encode_queries(self, queries: list[str], batch_size: int, **kwargs: Any):
        sentences = ["query: " + sentence for sentence in queries]
        emb = self.mdl.encode(sentences, batch_size=batch_size, **kwargs)
        return emb

    def encode_corpus(
        self, corpus: list[dict[str, str]], batch_size: int, **kwargs: Any
    ):
```
```suggestion
    def encode_queries(self, queries: list[str], batch_size: int = 32, **kwargs: Any):
        sentences = ["query: " + sentence for sentence in queries]
        emb = self.mdl.encode(sentences, batch_size=batch_size, **kwargs)
        return emb

    def encode_corpus(
        self, corpus: list[dict[str, str]], batch_size: int = 32, **kwargs: Any
    ):
```

**Comment by KennethEnevoldsen** [comment]
@Muennighoff will merge this in - and then feel free to start running it

**Pull request merged**